src/GraphQL.Analyzers.CodeFixes/GraphQL.Analyzers.CodeFixes.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IsPackable>false</IsPackable>
+    <RootNamespace>GraphQL.Analyzers</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.7.*" />
```

Comment:
Do you have any guidelines for version fixing?

Reply:
So far in this solutions we have fixed versions for all published packages and tests.  For sample projects, we let the version float like you show here.  Dependabot will create pull requests for each version bump, and we can ensure that the bump passes tests before merging.